vllm/model_executor/models/siglip.py

```diff
@@ -476,6 +488,10 @@ def __init__(
         num_hidden_layers_override: Optional[int] = None,
     ):
         super().__init__()
+        num_heads = config.num_attention_heads
+        tp_size = get_tensor_model_parallel_world_size()
+        self.shard_weight = USE_XFORMERS_OPS and num_heads % tp_size == 0
```

Comment:
What is this used for?

Reply:
This is used to bypass the fuse of qkv_proj on ViT for paligemma when using `Attention` from transformers, because we don't need to stack the weights of qkv_proj in this situation.

Reply:
Is it just me or do I not find the `shard_weight` attribute being used anywhere? Can you comment on the code with a reference to where this is used?